libs/redux/userTerrains/api.ts

```diff
@@ -0,0 +1,14 @@
+import clientApi from '../clientApi';
+
+const UserTerriansAPI = clientApi.injectEndpoints({
+  endpoints: (build) => ({
+    getuserTerrians: build.query({
+      query: () => `/user-terrain`,
+      providesTags: ['UserTerriansAPI'],
+    }),
+  }),
+});
+
+export const { useGetuserTerriansQuery } = UserTerriansAPI;
```

Comment:
Same sort of thing:

Should be `useGetUserTerrainsQuery` and `UserTerrainsApi`.